docs/develop/convert-javascript-to-typescript.md

```diff
@@ -78,13 +79,19 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 	declare var fabric: any;
 	```
 
-12. In the **Home.ts** file, change **'1.1'** to **1.1** (that is, remove the quotation marks) in the following line:
+12. In the **Home.ts** file, find the line `Office.initialize = function (reason) {` and add the following line immediately after it, as the first line inside the function:
+
+    ```typescript
+    (window as any).Promise = OfficeExtension.Promise;
+    ```
```

Comment:
I think this is fine, but for the sake of discussion, you it make more sense to frame this addition in context? Like,
```typescript
Office.initialize = function (reason) {
    // add the following line
    (window as any).Promise = OfficeExtension.Promise;
…
```

Reply:
Good suggestion, will do.